src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -465,8 +462,8 @@ private void removeTerritoriesThatArentWorthAttacking(final List<ProTerritory> p
     for (final Iterator<ProTerritory> it = prioritizedTerritories.iterator(); it.hasNext();) {
```

Comment:
Yep. Eventually will look to update to Java 8 streams in a lot of places in this class but outside the scope of the PR here.